tests/test_group_recruitment.py

```diff
@@ -0,0 +1,118 @@
+
+        now = datetime.datetime.utcnow()
+        venue.submission_stage = SubmissionStage(
+            double_blind=True,
+            due_date=now + datetime.timedelta(minutes = 30),
+            readers=[SubmissionStage.Readers.EVERYONE], 
+            withdrawn_submission_public=True, 
+            withdrawn_submission_reveal_authors=True, 
+            desk_rejected_submission_public=True,
+            force_profiles=False 
+        )
+
+        return venue
+    
+    def test_recruitment(self, venue, openreview_client, helpers):
+
+        venue.setup(program_chair_ids=['pc_venue23@mail.com'])
+        venue.create_submission_stage()
+        venue.group_builder.create_recruitment_committee_groups('Reviewers')
+
+        assert openreview_client.get_group('Venue.cc')
+        assert openreview_client.get_group('Venue.cc/Authors')
+        assert openreview_client.get_group('Venue.cc/Reviewers')
+        assert openreview_client.get_group('Venue.cc/Reviewers/Invited')
+        assert openreview_client.get_group('Venue.cc/Reviewers/Declined')
+        assert openreview_client.get_invitation('Venue.cc/Reviewers/Invited/-/Recruitment')
+        assert openreview_client.get_invitation('Venue.cc/Reviewers/Invited/-/Edit')
+
+        helpers.create_user('reviewer3@venue.cc', 'Reviewer', 'VenueThree')
+        
+        # use invitation to edit group content
+        openreview_client.post_group_edit(
+                invitation='Venue.cc/Reviewers/Invited/-/Edit',
+                group=openreview.api.Group(
+                    content = {
+                        'reduced_load': { 'value': [1,2,3] },
+                        'recruitment_template': { 'value': '''Dear {{fullname}},
+
+You have been nominated by the program chair committee of V 24 to serve as Reviewer. As a respected researcher in the area, we hope you will accept and help us make V 24 a success.
+
+You are also welcome to submit papers, so please also consider submitting to V 24.
+
+We will be using OpenReview.net and a reviewing process that we hope will be engaging and inclusive of the whole community.
+
+To respond the invitation, please click on the following link:
+
+{{invitation_url}}
+
+Please answer within 10 days.
+
+If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using. Visit http://openreview.net/profile after logging in.
+
+If you have any questions, please contact us at info@openreview.net.
+
+Cheers!
+
+Program Chairs
+''' },
+                        'recruitment_subject': { 'value': '[V 24] Invitation to serve as Reviewer' },
+                        'allow_overlap': { 'value': False }
+                    }
+                )
+            )
+        
+        invitee_details = '''reviewer1@venue.cc, Reviewer VenueOne\nreviewer2@venue.cc, Reviewer VenueTwo\n~Reviewer_VenueThree1'''
```

Comment:
So essentially you are expecting the users who are managing reviewer recruitment to provide a CSV file of users they want to invite. Do all the users in this list have to have OR profiles or is that optional? What about if the PC wants to remove someone from the invited group?